indico_jacow/controllers.py

```diff
@@ -8,11 +8,103 @@
 from collections import defaultdict
 from statistics import mean, pstdev
 
+from flask import session
+from sqlalchemy.orm import load_only
+from werkzeug.exceptions import Forbidden
+
 from indico.modules.events.abstracts.controllers.abstract_list import RHManageAbstractsExportActionsBase
-from indico.modules.events.abstracts.util import generate_spreadsheet_from_abstracts
+from indico.modules.events.abstracts.controllers.base import RHAbstractsBase
+from indico.modules.events.abstracts.models.review_ratings import AbstractReviewRating
+from indico.modules.events.abstracts.models.reviews import AbstractReview
+from indico.modules.events.abstracts.util import generate_spreadsheet_from_abstracts, get_track_reviewer_abstract_counts
+from indico.modules.events.management.controllers import RHManageEventBase
 from indico.util.spreadsheets import send_csv, send_xlsx
 from indico.web.flask.util import url_for
 
+from indico_jacow.views import WPAbstractsStats, WPDisplayAbstractsStatistics
+
+
+def _get_boolean_questions(event):
+    return [question
+            for question in event.abstract_review_questions
+            if not question.is_deleted and question.field_type == 'bool']
+
+
+def _get_question_counts(question, user):
+    counts = {}
+    for track in question.event.tracks:
+        query = (AbstractReviewRating.query
+                 .filter_by(question=question)
+                 .filter(AbstractReviewRating.value[()].astext == 'true')
+                 .join(AbstractReview)
+                 .filter_by(user=user, track=track))
+        counts[track] = query.count()
```

Comment:
You could probably avoid querying in a loop somewhat easily, but since this is just a plugin and statistics, I don't care *that* much.